### PR TITLE
feat(release): Add a new task to automatically chase last qa cards

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
Add a new task to automatically chase last qa cards

### Motivation
Automate release manager tasks

### CODE REVIEW

File: tasks/release.py [hunk 1]
```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
...
+@task
+def chase_for_qa_cards(_, version):
+    from collections import defaultdict
+
+    from slack_sdk import WebClient
+
+    cards = list_not_closed_qa_cards(version)
+    if len(cards) == 0:
+        print("No QA cards to chase", file=sys.stderr)
+        return
```
Comment (amenasria) [inline tasks/release.py:931]: **💬 suggestion:** ‏Could you instead use something like:
```suggestion
        Exit(color_message("No QA cards to chase", Color.RED), code=1)
```
With
```
from invoke.exceptions import Exit
from tasks.libs.common.color import Color, color_message
```

File: tasks/release.py [hunk 2]
```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
         client.chat_postMessage(channel=channel, text=message)
 
 
+@task
+def chase_for_qa_cards(_, version):
+    from collections import defaultdict
```
Comment (amenasria) [inline tasks/release.py:924]: **❓ question:** ‏For such 'small' imports couldn't we do it at the start of the file instead ?

**Review by amenasria — APPROVED** [review]
Left a few comments otherwise LGTM !

**Review by KevinFairise2 — APPROVED** [review]
Nice!

File: tasks/release.py [hunk 3]
```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
...
+
+    from slack_sdk import WebClient
+
+    cards = list_not_closed_qa_cards(version)
+    if len(cards) == 0:
```
Comment (FlorentClarret) [inline tasks/release.py:929]:
```suggestion
    if not cards:
```

nit

File: tasks/release.py [hunk 4]
```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
...
+        print("No QA cards to chase", file=sys.stderr)
```
Comment (FlorentClarret) [inline tasks/release.py:930]: Is it really an error? Having no cards sounds like a standard case to me

File: tasks/release.py [hunk 5]
```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
...
+        return
+    grouped_cards = defaultdict(list)
+    for card in cards:
+        grouped_cards[card["fields"]["project"]["key"]].append(card)
+    GITHUB_SLACK_MAP = load_and_validate("github_slack_map.yaml", "DEFAULT_SLACK_CHANNEL", DEFAULT_SLACK_CHANNEL)
+    GITHUB_JIRA_MAP = load_and_validate("github_jira_map.yaml", "DEFAULT_JIRA_PROJECT", DEFAULT_JIRA_PROJECT)
+    client = WebClient(os.environ["SLACK_API_TOKEN"])
+    print(f"Found {len(cards)} QA cards to chase", file=sys.stderr)
```
Comment (FlorentClarret) [inline tasks/release.py:938]: Is it an error?

File: tasks/release.py (hunk 1 above)
Comment (chouetz) [reply tasks/release.py:931]: `Exit` is an exception so I'm not sure about `Exit` itself. I'll propose something

**Pull request merged**